### ISSUE TITLE
Side Navigation does show up on Firefox for documentations

### ISSUE THREAD

**Issue by trojanh** [elixir-lang/ex_doc#801]
### Environment

* Browser: Firefox 57
* Operating system: Windows 10 Build 1703

### Current behavior

Side navigation for the topics does not show up on Firefox. 
For instance, if I open https://hexdocs.pm/ecto/Ecto.Changeset.html#delete_change/2 on Firefox it shows as follows

![firefox ecto docs bug](https://user-images.githubusercontent.com/22974490/33527132-6fd2383e-d871-11e7-94c5-2dbba3b20ade.PNG)

### Expected behavior

This navigation should be shown as it shows on Microsoft Edge and Google Chrome i.e 
![chrome docs](https://user-images.githubusercontent.com/22974490/33527170-f2531332-d871-11e7-9398-6618f35858df.PNG)

previously reported on Ecto repo
https://github.com/elixir-ecto/ecto/issues/2331

**Comment by josevalim** [comment]
Does it happen here? https://hexdocs.pm/ecto/2.2.7/Ecto.Changeset.html#delete_change/2

**Comment by josevalim** [comment]
I believe it was an outdated ExDoc version and the new Ecto version used latest ExDoc.

**Comment by josevalim** [comment]
I got a Windows computer and I was able to verify the sidebar works fine on the latest version, thanks.

**Comment by trojanh** [comment]
Hi @josevalim , really sorry for such delayed response. I was to able to verify that URL you posted above today right now and i see it still doesn't work. I am using Firefox version: 57.0.1 (64-bit).

**Comment by trojanh** [comment]
I debugged it and I was getting following error on browser console.
`Loading failed for the <script> with source “https://hexdocs.pm/ecto/dist/sidebar_items-5d15d2a288.js”.
Ecto.Changeset.html:13
ReferenceError: sidebarNodes is not defined
[Learn More]
app-778c9ef903.js:4:25872
Source map error: request failed with status 404
Resource URL: https://hexdocs.pm/ecto/dist/app-778c9ef903.js
Source Map URL: app-778c9ef903.js.map`

After investigating I found that this script was blocked by Adblock add on on Firefox. Thanks @josevalim for looking into this. I will be extra careful with websites not loading correctly when using this particular add-on.
